### PR TITLE
Feature/us941721 schema update

### PR DESCRIPTION
**Pull request by uppoju**
Update schema and graphql queries for the new entities
- FederatedGroups
- FederatedUsers
- FederatedIdps
- L7Policy
- L7Service
- LdapIdp
- SimpleLdapIdp
- PolicyBackedIdp

Removed deprecated types from the graphql queries

### CODE REVIEW

File: modules/graphman-bundle.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 const utils = require("./graphman-utils");
 const SCHEMA_METADATA = require("./graphman").schemaMetadata();
-const GOID_PLURAL_METHODS = ["fipUsers", "internalUsers", "fipGroups", "internalGroups", "ldaps", "fips", "trustedCerts"];
+const GOID_PLURAL_METHODS = ["federatedUsers", "internalUsers", "federatedGroups", "internalGroups", "ldapIdps", "federatedIdps", "trustedCerts"];
```
Comment (graju256) [inline modules/graphman-bundle.js:4]: I think, we should be supporting the deprecated entity types as part of dynamic query.

File: queries/mutation.gql [hunk 2]
```diff
@@ -1,27 +1,23 @@
 mutation installBundle (
     $activeConnectors: [ActiveConnectorInput!]!=[],
     $administrativeUserAccountProperties: [AdministrativeUserAccountPropertyInput!]!=[],
-    $backgroundTaskPolicies:[BackgroundTaskPolicyInput!]!=[],
```
Comment (graju256) [inline queries/mutation.gql:4]: I hope, you pulled this content from UneasyRooster!

Comment (uppoju) [reply queries/mutation.gql]: updated with uneasyrooster file

File: modules/graphman-bundle.js (hunk 1 above)
Comment (uppoju) [reply modules/graphman-bundle.js]: added deprecated entity types

**Review by graju256 — APPROVED** [review]
LGTM

**Pull request merged**